maven-surefire-common/src/test/java/org/apache/maven/plugin/surefire/AbstractSurefireMojoToolchainsTest.java

```diff
@@ -47,7 +49,7 @@
  * Test for {@link AbstractSurefireMojo}. jdkToolchain parameter
  */
 @RunWith( PowerMockRunner.class )
-@PrepareForTest( {AbstractSurefireMojo.class} )
+@PrepareForTest( {AbstractSurefireMojo.class, DefaultJavaToolChain.class} )
```

Comment:
for your info, the annotation `PrepareForTest` is used with a class which is mocked using the method `mockStatic`. This does not happen in your new test method `shouldChangeJavaHomeFromToolchain`.

Reply:
Right, I tried following your advice but couldn't figure out how to do it that way.  I'll remove this.